tests/drivers/gpio/gpio_nrf/src/main.c

```diff
@@ -58,4 +58,37 @@ ZTEST(gpio_nrf, test_gpio_high_drive_strength)
 		err);
 }
 
+/*
+ * Nordic Semiconductor specific,
+ * gpio manipulation with disabled NRFX interrupts
+ */
+ZTEST(gpio_nrf, test_gpio_manipulation_nrfx_int_disabled)
+{
+	int response;
+	const struct device *port;
+
+	port = DEVICE_DT_GET(TEST_NODE);
+	zassert_true(device_is_ready(port), "GPIO dev is not ready");
+
+	response = gpio_pin_configure(port, TEST_PIN, GPIO_PUSH_PULL | GPIO_ACTIVE_HIGH);
+	zassert_ok(response, "Pin configuration failed: %d", response);
+
+	response = gpio_pin_set(port, TEST_PIN, 0);
+	zassert_ok(response, "Pin low state set failed: %d", response);
+
+	response = gpio_pin_set(port, TEST_PIN, 0);
+	zassert_ok(response, "Pin hight state set failed: %d", response);
+
+	response = gpio_pin_toggle(port, TEST_PIN);
+	zassert_ok(response, "Pin toggle failed: %d", response);
+
+	response = gpio_pin_configure(port, TEST_PIN, GPIO_INPUT | GPIO_PULL_DOWN);
+
+	response = gpio_pin_get(port, TEST_PIN);
+	zassert_equal(response, 0, "Invalid pin state: %d", response);
+
+	response = gpio_pin_interrupt_configure(port, TEST_PIN, GPIO_INT_ENABLE | GPIO_INT_HIGH_1);
+	zassert_equal(response, -ENOSYS);
```

Comment:
Isn't this going to fail when run in `drivers.gpio.gpio_nrf` subcase? (With `GPIO_NRFX_INTERRUPT` enabled.)

Reply:
Moved to prj.conf as this setting does not affect the first test case.